demo_nodes_py/demo_nodes_py/topics/listener_serialized.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import String
+
+
+class SerializedSubscriber(Node):
+
+    def __init__(self):
+        super().__init__('serialized_subscriber')
+        self.subscription = self.create_subscription(
+            String,
+            'chatter',
+            self.listener_callback,
+            raw=True)  # We're subscribing to the serialized bytes, not std_msgs.msg.String
+        self.subscription  # prevent unused variable warning
```

Comment:
it looks like we don't need that in our other python demos. 
https://github.com/ros2/demos/blob/master/demo_nodes_py/demo_nodes_py/topics/listener.py#L25
Where did you encounter this warning?

Reply:
Ah I actually copied this from the rclpy examples. It can probably be removed.